File: data-client/src/main/java/org/wikipedia/login/LoginOAuthResult.java
package org.wikipedia.login;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import org.wikipedia.dataclient.WikiSite;

public class LoginOAuthResult extends LoginResult {

    public LoginOAuthResult(@NonNull WikiSite site, @NonNull String status, @Nullable String userName,
                     @Nullable String password, @Nullable String message) {
        super(site, status, userName, password, message);
    }
}

File: app/src/main/java/fr/free/nrw/commons/auth/login/LoginOAuthResult.kt
package fr.free.nrw.commons.auth.login

import org.wikipedia.dataclient.WikiSite

class LoginOAuthResult(
    site: WikiSite,
    status: String,
    userName: String?,
    password: String?,
    message: String?
) : LoginResult(site, status, userName, password, message)
